pathtools/glob.go

```diff
@@ -52,11 +38,15 @@ func Glob(pattern string) (matches, dirs []string, err error) {
 // In general ModuleContext.GlobWithDeps or SingletonContext.GlobWithDeps
```

Comment:
Another copy of GlobWithExcludes two lines up (but github won't let me comment on those)